main.go

```diff
@@ -104,7 +106,11 @@ func main() {
 		if *shortFlag {
 			flagsCommands = flagsCommands + " -short"
 		}
-		coverageCommand = fmt.Sprintf("%s %s && %s", testCommand, flagsCommands, toolCoverCommand)
+		if *rootFolderFlag != "" {
+			coverageCommand = fmt.Sprintf("go test %s/... -coverprofile=coverage.out %s && %s", *rootFolderFlag, flagsCommands, toolCoverCommand)
```

Comment:
Somewhat differs in style from original approach, but seems reasonable enough.

Reply:
You have that first part as a constant `go test ./... -coverprofile=coverage.out` do you think it should get changed to be a variable or something?

Reply:
Yea, you make a good point, I'm not even sure there's a good reason fo rthe command to be stored in a variable in the first place (maybe I had something in mind when it was first written, but I've since forgotten.
The else condition line could probably just be changed to the following, and it would probably look cleaner anyway, what do you think?
```
coverageCommand = fmt.Sprintf("go test ./... -coverprofile=coverage.out %s && %s", testCommand, flagsCommands, toolCoverCommand)
```

Reply:
I went ahead and removed that `testCommand` const and simplified the `coverageCommand` like so, such that the starting directory defaults to current `.` or it gets set to whatever comes in on the flag

```
		rootFolder := "."
		if *rootFolderFlag != "" {
			rootFolder = *rootFolderFlag
		}
		coverageCommand = fmt.Sprintf("go test %s/... -coverprofile=coverage.out %s && %s", rootFolder, flagsCommands, toolCoverCommand)
```

Also, how are you writing tests for the `main` function anyway? Do you have any guidance for that or do you think this is simple enough we could proceed as it looks like you have been for the other flags? Might be a good idea to split the `main` into smaller functions that can be tested in isolation. What do you think?